sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceAsyncClientBuilder.java

```diff
@@ -0,0 +1,19 @@
+package com.azure.data.tables;
+
+public class TableServiceAsyncClientBuilder {
```

Comment:
The naming is incorrect https://azure.github.io/azure-sdk/java_design.html#service-clients